src/libraries/Common/tests/System/Net/Http/HttpProtocolTests.cs

```diff
@@ -512,6 +517,12 @@ public async Task GetAsync_Chunked_VaryingSizeChunks_ReceivedCorrectly(int maxCh
         [InlineData("head", "HEAD")]
         [InlineData("post", "POST")]
         [InlineData("put", "PUT")]
+        [InlineData("delete", "DELETE")]
```

Comment:
Moved from [`SocketsHttpHandler_HttpProtocolTests`](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Net.Http/tests/FunctionalTests/SocketsHttpHandlerTest.cs#L113-L119) so it can be shared.